### ISSUE TITLE
Regression: Jeremy McGrath's Offroad [NPUB30368] now freezes on loading menus (#6293)

### ISSUE THREAD

**Issue by legend800** [RPCS3/rpcs3#6681]
Game freezes when trying to load the menu. Game was fully playable prior to this regression.

Last call logged:
E {PPU[0x1000000] Thread (main_thread) [0x005a14a0]} 'cellAudioCreateNotifyEventQueue' failed with 0x80010014 : CELL_EEXIST [1]

[McGrath.log.gz](https://github.com/RPCS3/rpcs3/files/3668560/McGrath.log.gz)

Config:
Windows 10 64-bit
GeForce GTX 1080
i7-6700k @ 4GHZ
16GB RAM

**Comment by elad335** [comment]
not an error, rename issue.

**Comment by legend800** [comment]
Broken by PR:  https://github.com/RPCS3/rpcs3/issues/6681

Fine on previous build (8209):
[8209.log.gz](https://github.com/RPCS3/rpcs3/files/3668794/8209.log.gz)

**Comment by legend800** [comment]
@scribam 
Just a quick update, this game is still broken on latest build from today.

**Comment by legend800** [comment]
Game is still broken with the same behavior. Game was fully playable and hasn't been able to load for about 8 months now because of the PR i listed above.

**Comment by elad335** [comment]
Maybe retry to tracing the offending pr, give before and after logs as well.
It's unlikely that it's the currently mentioned pr.

**Comment by legend800** [comment]
Yep, you're right. After an hour of testing and trying various builds, I finally figured out what's going on.

First, this game has always been a little flaky on boot, for years. That's why I think I got the builds wrong the first time. Basically before being broken (8437 and prior), it would load 8/10 times. That went to 1/35 (tried it at least that many) on build 8349. I checked several builds prior to those builds and after those and all exhibited consistent behavior, so I'm  sure it started with 8349. 

This is the PR that broke it:
Add fallback implementation for waitable atomics
https://github.com/RPCS3/rpcs3/pull/6293
@Nekotekina 

Here's the page where build/prs are at the time the regression occurred:
https://rpcs3.net/compatibility?b&p=34

Build 8437 log (works):
[8437.gz](https://github.com/RPCS3/rpcs3/files/4452671/8437.gz)

Build 8449 log (broken):
[8349.gz](https://github.com/RPCS3/rpcs3/files/4452673/8349.gz)

**Comment by legend800** [comment]
Update:
I can now boot this game consistently (>90%) due to recent changes I believe. It's still very timing sensitive and requires TSX off to boot reliably. Needs logging off and LLVMs also to boot properly. 

Unfortunately, it now just always hang right before loading the main menu on this screen. Toggled every possible setting, cleared cache, save, etc. but can almost never gets past this screen.
![2020-08-15 12_30_46-Window](https://user-images.githubusercontent.com/20983844/90321334-818fd280-defd-11ea-86d2-048ab6d3d86c.png)

Got in-game twice out of 40+ loads:
![2020-08-15 13_06_31-Window](https://user-images.githubusercontent.com/20983844/90321340-8f455800-defd-11ea-80f6-63becd49a4fa.png)

**Comment by legend800** [comment]
[Jeremy-RPCS3.log](https://github.com/RPCS3/rpcs3/files/5898719/Jeremy-RPCS3.log)

Now I can't get into the menus anymore. Numerous restarts + diff configs but nothing now. Worked perfectly in 2019.

**Comment by legend800** [comment]
Game still does not boot as of 5/15/21 build. 

[RPCS3.log](https://github.com/RPCS3/rpcs3/files/6483959/RPCS3.log)

**Comment by legend800** [comment]
Still active as of 0.0.16-12427.

![2021-06-28 00_00_57-FPS_ 60 00 _ Vulkan _ 0 0 16-12427 _ Jeremy McGrath's Offroad  NPUB30368](https://user-images.githubusercontent.com/20983844/123595143-650f2880-d7a5-11eb-9e69-5646e8c215b0.png)

**Comment by legend800** [comment]
Still broken as of 1/30/22. Attached latest log. 

[RPCS3.log.gz](https://github.com/RPCS3/rpcs3/files/7967642/RPCS3.log.gz)

**Comment by legend800** [comment]
Activating HLE libsail.sprx gets the game bootable and in-game again, but videos are missing and there's a lot of black menu screens making it unplayable atm. Some screens:
https://discord.com/channels/272035812277878785/272081036316377088/937625595481837618

**Comment by legend800** [comment]
Still an issue as of 0.24.14141 (Sep '22).

**Comment by elad335** [comment]
Maybe update logs with SPU/PPU debug options enabled.

**Comment by legend800** [comment]
@elad335 Sure, here's a log with those enabled. As noted above, it seems libsail.sprx related.

[RPCS3.log.gz](https://github.com/RPCS3/rpcs3/files/9541502/RPCS3.log.gz)

**Comment by digant73** [comment]
this issue is fixed by PR #19562
